.github/workflows/action.yml

```diff
@@ -5,7 +5,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-elixir@v1
+      - uses: erlef/setup-elixir@v1.7
```

Comment:
Thanks for updating this! Will probably have to list the Elixir version that we're testing against on the README